widget-src/Widget.tsx

```diff
@@ -13,7 +14,8 @@ function Widget() {
   const [showDescription, setShowDescription] = useSyncedState('showDescription', true);
   const [showStatus, setShowStatus] = useSyncedState('showStatus', '0');
   const [showVersion, setShowVersion] = useSyncedState('showVersion', false);
-  const [showBranding, setShowBranding] = useSyncedState('showBradning', true);
+  const [showBranding, setShowBranding] = useSyncedState('showBranding', true);
```

Comment:
Does changing this, ugh my fat fingers, possibly introduce this setting breaking on older logs?

Reply:
I don't think so, did you try turning the branding on/off for the ones that you created from `main`?
It is one of those things that I was actually confused was working in the first place because you correctly spelled the variable everywhere except that spot.